lib/aweplug/middleware/statuslogger.rb

```diff
@@ -0,0 +1,19 @@
+module Aweplug
+  module Middleware
+
+    class StatusLogger < Faraday::Middleware
```

Comment:
Would you please add some documentation like in some of the other classes about what this does?